dal/daltest/helpers.go

```diff
@@ -17,3 +18,42 @@ func InitTestDB(t *testing.T) *dal.DB {
 	assert.NilError(t, err)
 	return db
 }
+
+// GetHashBytes creates a byte array from a hash string.
+func GetHashBytes(t *testing.T, hashString string) []byte {
```

Comment:
If we don't use this function outside package, I think we don't need to export this. Let's make this a private function.

Reply:
we actually use it in different packages. eg. dal_test